packages/contracts/test/exchange/helpers.ts

```diff
@@ -38,12 +40,31 @@ describe('Exchange', () => {
             deployer.deployAsync(ContractName.DummyToken, constants.DUMMY_TOKEN_ARGS),
             deployer.deployAsync(ContractName.DummyToken, constants.DUMMY_TOKEN_ARGS),
         ]);
+        const assetProxyDispatcher = await deployer.deployAsync(ContractName.AssetProxyDispatcher);
```

Comment:
Do we actually need to deploy and configure these contracts for the purposes of these tests?